roles/contiv/templates/contiv.cfg.j2

```diff
@@ -0,0 +1,6 @@
+{
+  "K8S_API_SERVER": "https://{{ hostvars[groups['masters'][0]]['ansible_' + netmaster_interface].ipv4.address }}:{{ kube_master_api_port }}",
```

Comment:
Agreed. I tested earlier by using "openshift.default.svc.cluster.local" and other such options in the above j2 template but those didnt seem to work (possibly because contiv needs to be able to access the kubernetes api right in the beginning when even DNS is not reachable or has not populated these domain name entries into the local DNS yet). Hence I reverted to using the IP address with the expectation of a VIP based solution for HA (keepalived/ ucarp/ pacemaker etc). Agreed that this limits our ability to load balance the API server for now but at least we know it works reliably in our testing. Could we visit the load balancing after this merge so that we determine a solution that works reliably in our testing and at that time update this j2 template if needed ? In any case someone can always over-ride this variable through the inventory file if they have a scenario that works reliably with using a cluster DNS name rather than IP address.